data/transactions/logic/assembler.go

```diff
@@ -1649,6 +1689,16 @@ func (ops *OpStream) assemble(text string) error {
 		}
 	}
 
+	// Before back branches, the pseudo ops can and do complain during first pass
+	if ops.Version >= backBranchEnabledVersion {
+		if ops.hasIntcBlock && ops.hasPseudoInt {
+			ops.error("program has int instruction with a manual intcblock")
+		}
+		if ops.hasBytecBlock && ops.hasPseudoByte {
+			ops.error("program has byte instruction with a manual bytecblock")
+		}
+	}
+
```

Comment:
@jannotti Sanity checking my understanding:  From my read, it appears `asmByte` and `asmInt` _prevent_ the highlighted block from executing by returning `pushint` / `pushbyte`.  

If my read is accurate, I'm wondering:
* Is it safe to remove the highlighted check?
* Alternatively, since the unit tests pass with the highlighted stanza deleted, is there a test case missing?

```suggestion
```

Reply:
They only use pushint in recent versions.  In pre-backbranch versions,  they just complain (in `IntLiteral` for example).

Test incoming, after lunch.

I was thinking that pushint was introduced at the same time as backbranches, but pushint came sooner, so maybe we can make it use pushint in v3 as well. I need to recall why I was focused on back branches. I think it's because before back branches you could be more sure of what's in the cblocks. But even there, conditionals cause trouble.

Reply:
Ok, I've convinced myself you are correct that we can remove the check.  I believe the combination of the checks in asmInt/Byte and asmIntcblock/asBytecblock take care of this. Whichever happens second complains.